src/lt/plugins/clojure.cljs

```diff
@@ -537,12 +591,20 @@
       (string/replace #"__\|(.*)\|__" watch)))
 
 (behavior ::cljs-watch-custom-src
+          :desc "Clojurescript: Prepare exp for watching"
```

Comment:
We should avoid using aliases like exp until we have a common place for users to look for commonly used aliases. exp -> expression should work